charts/librespeed/README.md

```diff
@@ -0,0 +1,96 @@
+| `service.loadBalancerSourceRanges` | List of IP CIDRs allowed access to load balancer (if supported)          | None                      |
+| `service.externalTrafficPolicy`    | Set the externalTrafficPolicy in the Service to either Cluster or Local  | `Cluster`                 |
+| `ingress.enabled`                  | Enables Ingress                                                          | `false`                   |
+| `ingress.annotations`              | Ingress annotations                                                      | `{}`                      |
+| `ingress.path`                     | Ingress path                                                             | `/`                       |
+| `ingress.hosts`                    | Ingress accepted hostnames                                               | `chart-example.local`     |
+| `ingress.tls`                      | Ingress TLS configuration                                                | `[]`                      |
+| `persistence.enabled`              | Use persistent volume to store data                                      | `false`                   |
+| `persistence.size`                 | Size of persistent volume claim                                          | `5Gi`                     |
+| `persistence.existingClaim`        | Use an existing PVC to persist data                                      | `nil`                     |
+| `persistence.storageClass`         | Type of persistent volume claim                                          | `-`                       |
+| `persistence.accessModes`          | Persistence access modes                                                 | `ReadWriteOnce`           |
+| `persistence.subPath`              | Mount a sub dir of the persistent volume                                 | `nil`                     |
+| `resources`                        | CPU/Memory resource requests/limits                                      | `{}`                      |
+| `nodeSelector`                     | Node labels for pod assignment                                           | `{}`                      |
+| `tolerations`                      | Toleration labels for pod assignment                                     | `[]`                      |
+| `affinity`                         | Affinity settings for pod assignment                                     | `{}`                      |
+| `podAnnotations`                   | Key-value pairs to add as pod annotations                                | `{}`                      |
+| `deploymentAnnotations`            | Key-value pairs to add as deployment annotations                         | `{}`                      |
+
+Specify each parameter using the `--set key=value[,key=value]` argument to `helm install`. For example,
+
+```shell
+helm install --name my-release \
+  --set config.timezone="America/New_York" \
+    billimek/Librespeed
```

Comment:
This is really minor, but can we make this instance lowercase (e.g. `librespeed`)